docs/features/tag-discovery/spec.md

```
@@ -0,0 +1,427 @@
+  Future<List<Event>> getPartiesByTag(String tagId, {int limit = 10, int offset = 0});
+  Future<List<Event>> getTagRecommendations({int limit = 10});
+  Future<List<Tag>> searchTags(String query);
+  Future<void> upsertUserInterestTags(List<String> tagIds);
+}
+```
+
+### 4. 유저 앱 UI
+
+#### 4.1 홈 피드 상단: 인기 태그 칩바
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)